testsuite/running-tests.md

```diff
@@ -5,55 +5,33 @@ This document will enumerate everything you need to do run the sonobuoy based co
-5. Download and install [az cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
-6. Download and install [python 3](https://www.python.org/downloads)
 
-### Additional Prerequisites for Arc enabled Data Services
-
-1. Install [arcdata extension](https://docs.microsoft.com/en-us/azure/azure-arc/data/release-notes).
 
 ## Running the script and publishing the results
 
-1. Clone this repository.
-2. Edit the [`partner-metadata.md`](partner-metadata.md) file and fill in the required details. For reference, please see the [`partner-metadata-sample.md`](partner-metadata-sample.md) file.
-
-### Arc enabled Kubernetes
-1. Edit the [`k8s-conformance-test-suite.sh`](k8s-conformance-test-suite.sh) file and set the values for the required environment variables.
-2. If your cluster is behind an outbound proxy, please edit the above file according to the instruction provided as comments for proxy configuration.
-3. Make the test suite file executable by running `chmod +x k8s-conformance-test-suite.sh`.
-4. Execute the script by running `./k8s-conformance-test-suite.sh`.
-5. The test suite will take the storage account details as environment variables and will handle publishing the results in the right format.
-
-*The above script is **bash only**. Please use the [`k8s-conformance-test-suite.ps1`](k8s-conformance-test-suite.ps1) script for windows hosts.
+1. Clone three files azure-arc-conformance.properties, launch.yaml and kustomization.yaml.
+2. Create a directory i.e., conformanceTest and move azure-arc-conformance.properties, launch.yaml and kustomization.yaml to the directory
+3. Edit the azure-arc-conformance.properties file for enabling test plugins.
+4. `cd conformanceTest`
+5. Run command `kubectl apply -k .` Now tests should automatically start running.
 
-### Arc enabled Data Services
-1. Edit the [`ds-conformance-test-suite.sh`](ds-conformance-test-suite.sh) file and set the values for the required environment variables.
-2. If your cluster is behind an outbound proxy, please edit the above file according to the instruction provided as comments for proxy configuration.
-3. If you wish to bring your own custom control deployment profile with your own configuration, please follow the below process to provide the `control.json` to sonobuoy plugin.
-```
-az arcdata dc config init --source azure-arc-aks-default-storage --path /tmp/dcconfig
-kubectl create ns arc-ds-config ; kubectl -n arc-ds-config create configmap arc-ds-config --from-file=/tmp/dcconfig/control.json
-```
-Please update the `CONFIG_PROFILE` variable in the above script accordingly.
 
-4. Make the test suite file executable by running `chmod +x ds-conformance-test-suite.sh`.
-5. Execute the script by running `./ds-conformance-test-suite.sh`.
-6. The test suite will take the storage account details as environment variables and will handle publishing the results in the right format.
+### Watching kubernetes pod for current running status
```

Comment:
Add the changes for troubleshooting, retrieving results and cleaning up the cluster under the Arc enabled Kubernetes section. Move the existing retrieving results and cleaning up the the cluster details under data services.

Reply:
ack